src/NzbDrone.Core/Parser/LanguageParser.cs

```diff
@@ -14,7 +14,7 @@ public static class LanguageParser
 
         private static readonly RegexReplace[] CleanSeriesTitleRegex = new[]
             {
-                new RegexReplace(@".*?\.(S\d{2}(?:E\d{2,4})*\..*)", "$1", RegexOptions.Compiled | RegexOptions.IgnoreCase)
+                new RegexReplace(@".*?(?:\.|\s)(S\d{2}(?:E\d{2,4})*(?:\.|\s).*)", "$1", RegexOptions.Compiled | RegexOptions.IgnoreCase)
```

Comment:
`[. ]` instead of `(?:\.|\s)` is our preferred way to, but it'd be worth adding `_` as well (`[_. ]`)